UI/FleetWnd.cpp

```diff
@@ -3003,13 +3047,18 @@ void FleetWnd::Refresh() {
         if (!fleet)
             continue;
 
+        auto fleet_loc = GG::Pt(GG::X(fleet->X()), GG::Y(fleet->Y()));
+        // Grow the fleets bounding box
+        fleets_bounding_box = CreateOrGrowBox(fleets_that_exist.empty(), fleets_bounding_box, fleet_loc);
+
         fleets_that_exist.insert(fleet_id);
-        fleet_locations_ids.insert({{fleet->SystemID(), GG::Pt(GG::X(fleet->X()), GG::Y(fleet->Y()))},
-                                    fleet_id});
+        fleet_locations_ids.insert({{fleet->SystemID(), fleet_loc}, fleet_id});
 
     }
+    GG::Pt bounding_box_center{fleets_bounding_box.MidX(), fleets_bounding_box.MidY()};
```

Comment:
might make your initialization format a bit more consistent... here `GG::Pt b_b_c{}`, above `auto f_l = GG::Pt()`.

Reply:
Fixed in two locations.